worker/job-handlers.ts

```diff
@@ -256,6 +257,12 @@ export async function scheduleJob(job: WorkerJob, chainId: string) {
         case 'sync-vebal-totalSupply':
             await runIfNotAlreadyRunning(job.name, chainId, () => veBalService.syncVeBalTotalSupply(), 0.01);
             break;
+        case 'sync-vebal-voting-list':
+            //QUESTION: does it make sense to split it by chainId??
+            // TODO: define a proper sampling rate
+            const samplingRate = 0.01;
+            await runIfNotAlreadyRunning(job.name, chainId, () => votingListService.syncVotingList(), samplingRate);
```

Comment:
The voting list is a feature that only works in MAINNET, however, in order to create the voting list we need to fetch info from all the L2 subgraphs. 
I designed it in a way that you only run it once and it writes the rows for all the networks. 
Following your typical job-handler flow,  we could easily split the single process in 1 process per network but, I see these trade-offs:

On one hand, having one job per network would make debugging a bit simpler: if the process fails in a concrete network we know which concrete network service to fix.
On the other hand, monitoring as much crons as networks can be extra cognitive/resource load that we don't really need.  

I'm sure that you have better arguments after your experience with the system, so happy to read them.